packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -87,6 +87,16 @@ library LibProving {
                     ) revert L1_NOT_ORACLE_PROVER();
                 }
             }
+
+            if (
+                config.realProofSkipSize > 1 &&
+                blockId % config.realProofSkipSize != 0
+            ) {
+                // For this block, real ZKP is not necessary, so the oracle
+                // proof will be treated as a real proof (by setting the prover
+                // to a non-zero value)
+                evidence.prover = address(1);
```

Comment:
Well, the mechanics heavily depends on the proof time. In case we will be able to find a good parameter (kinda average) it could work yes!
Depends also what is the difference between 'real' proof time? 
30 minutes vs. the dummy ones (12-60s) ? If so, it might be very tricky to make it work. If the diff is not that drastic, i assume we need to find the sweet spot (=very close to the average).
Soon pushing the simulation code, you can try, but obviously it does not have this scenario possible. (Can try to incorporate tho next week to see how it behaves within these new circumstences)

Reply:
Any reason not to submit these at the expected proof time? It's only for testnets so seems good enough even if we could manipulate the prover fee.

Reply:
We can certain try to submit these proofs with a roughly average proof delay, but I just want to understand the nature of the impact of such a really small proof delay.

Reply:
If most proofs keep being submitted below the target time the prover fee/reward will go to zero (except if the actual proof being submitted is a LOT longer than the target time to try and keep in balance somehow, but that would be very suboptimal)

Reply:
https://github.com/taikoxyz/taiko-mono/pull/13657

Reply:
@davidtaikocha please make sure you don't submit these oracle proofs too soon!

Reply:
@dantaik It can come quick but would be good to have them as 'strictly' automated as possible so that we can calculate the rest properly (e.g.: every 10th block comes somewhere 1-2mins)